win32ss/user/ntuser/desktop.c

```diff
@@ -2368,7 +2368,7 @@ IntCreateDesktop(
     Status = ObReferenceObjectByHandle(hDesk,
                                        0,
                                        ExDesktopObjectType,
-                                       KernelMode,
+                                       AccessMode,
```

Comment:
Why? The hDesk handle was obtained from kernel mode, via the previous call; it didn't come from e.g. user mode.

Reply:
> always use `UserMode` here

That would currently work, but not match the rest of current code either, before and after this call.

> try this

No thanks: my goal is to fix, at least improve, current code, not to guess what future calls with `KernelMode` you are planning would look like.
That should be checked, as the rest of the code, when there is an actual case.

Reply:
You are making a change and Hermès says it may not be correct, and suggests a fix. It seems to be a response of "No thanks" means "please close this PR, I'm not interested in making it correct".

Reply:
> not interested

Not at all: I merely replied that 'try this' about some future call is too vague (for me) as 'an actual [test]case'...

Reply:
@HBelusca, could you either accept this PR as is, or give an explicit testcase? Thanks.